README.md

```diff
@@ -274,11 +274,12 @@ Additional lists you might find useful:
 ## REST and API
 *Plugins and web tools for developing REST-ful APIs.*
 
+- [Alt3/Swagger plugin](https://github.com/alt3/cakephp-swagger) - Swagger plugin for documenting your CakePHP 3.x APIs.
```

Comment:
3.x should be removed IMO
Also the plugin is doubled:

    For documenting your CakePHP APIs.

Or sth?